tests/integrations/realcluster/etcd_key_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2024 TiKV Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package realcluster
+
+import (
+	"fmt"
+	"slices"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"github.com/tikv/pd/client/utils/testutil"
+)
+
+type etcdKeySuite struct {
+	clusterSuite
+}
+
+func TestEtcdKey(t *testing.T) {
+	suite.Run(t, &etcdKeySuite{
+		clusterSuite: clusterSuite{
+			suiteName: "etcd_key",
+		},
+	})
+}
+
+func TestMSEtcdKey(t *testing.T) {
+	suite.Run(t, &etcdKeySuite{
+		clusterSuite: clusterSuite{
+			suiteName: "etcd_key",
+			ms:        true,
+		},
+	})
+}
+
+var (
+	// The keys that prefix is `/pd`.
+	pdKeys = []string{
+		"",
+		"/pd//alloc_id",
+		"/pd//config",
+		// If not call `UpdateGCSafePoint`, this key will not exist.
+		// "/pd//gc/safe_point",
+		"/pd//gc/safe_point/service/gc_worker",
+		"/pd//keyspaces/id/DEFAULT",
+		"/pd//keyspaces/meta/",
+		"/pd//leader",
+		"/pd//member//binary_version",
+		"/pd//member//deploy_path",
+		"/pd//member//git_hash",
+		"/pd//raft",
+		"/pd//raft/min_resolved_ts",
+		"/pd//raft/r/",
+		"/pd//raft/s/",
+		"/pd//raft/status/raft_bootstrap_time",
+		"/pd//region_label/keyspaces/",
```

Comment:
Can you explain the difference between `//` and `/`?

Reply:
To ensure consistency in each test, I removed the numerical parts in the tests.

Reply:
<img width="789" alt="image" src="https://github.com/user-attachments/assets/5bc39775-e708-47a3-9c3a-4d43e8979caa">

Reply:
I'm afraid it's hard to maintain later.

Reply:
Listing them one by one is somewhat challenging, but it does have two main benefits:  
1. It adds tests for them.  
2. Through testing, we also clearly understand in which scenarios different keys will be generated.

Moreover, I really haven't thought of any better way to achieve it. - -